From 73d2af69f280cf58296877cdab22a952dcd49307 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sat, 18 Apr 2020 07:12:00 +0300
Subject: [PATCH] [SW-0] goya2: sbtool should not use arc macros

First reason since they are not needed
Second they cause many compilation warnings:
warning: "ELPFAST_TYPE" redefined

Change-Id: I2b072b476fd682a71355b7c4d1327d69cbe4d748
---

diff --git a/Makefile b/Makefile
index 2759833..795056b 100644
--- a/Makefile
+++ b/Makefile
@@ -27,6 +27,7 @@
 ARCBOOT_CFLAGS += -DBTLID=0x30
 ARCBOOT_CFLAGS += -DFLASH_START_OFFSET=0x60000
 
+ifneq ($(CROSS_COMPILE),)
 LDFLAGS = -flto
 LDFLAGS += -L $(LIBDIR) -lsb -lellipsys -L $(HL_DEV)
 LDFLAGS += -T $(HL_DEV)/$(HL_TARGET)_$@.lds
@@ -47,6 +48,7 @@
 ARCBOOT_ROM_SRCS_C := app/arcboot_rom.c
 ARCBOOT_ROM_SRCS_S := app/crt0_rom.S
 ARCBOOT_ROM_OBJECTS := $(ARCBOOT_COMMON_SRCS_C:.c=.o) $(ARCBOOT_ROM_SRCS_C:.c=.o) $(ARCBOOT_ROM_SRCS_S:.S=.o)
+endif
 
 .PHONY: all libsb libellipsys sbtool
 all: libsb libellipsys
